bump_version

```diff
@@ -0,0 +1,189 @@
+    last_master_merge_commit="$(
+        git log --merges --pretty=oneline \
+            | grep -F "${last_master_branch}" \
+            | awk '{ print $1 }' \
+            || true # Don't die here with errexit
+    )"
+    if [[ -z ${last_master_merge_commit} ]]; then
+        # This repo has no last proper commit, so just skip it
+        popd
+        continue
+    fi
+    # Get all the PR merge commits since the last master merge commit in `jellyfin`
+    pr_merges_since_last_master+=( $(
+        git log --merges --pretty=oneline ${last_master_merge_commit}..HEAD \
+            | grep -F "Merge pull request" \
+            | awk '{ print $1 }'
+    ) )
+
+    for commit_hash in ${pr_merges_since_last_master[@]}; do
+        git_show_details="$( git show ${commit_hash} )"
+        pull_request_id="$(
+            awk '
+                /Merge pull request/{ print $4 }
+                { next }
+            ' <<<"${git_show_details}"
+        )"
+        pull_request_description="$(
+            awk '
+                /^[a-zA-Z]/{ next }
+                /^    Merge/{ next }
+                /^$/{ next }
+                { print $0 }
+            ' <<<"${git_show_details}"
+        )"
+        pull_request_description="$( sed ':a;N;$!ba;s/\n//g; s/  \+//g' <<<"${pull_request_description}" )"
+        changelog_strings_repo_github="${changelog_strings_repo_github}\n* ${pull_request_id}: ${pull_request_description}"
+        changelog_strings_repo_deb="${changelog_strings_repo_deb}\n  * $( sed 's/#/PR/' <<<"${pull_request_id}" ) ${pull_request_description}"
+        changelog_strings_repo_yum="${changelog_strings_repo_yum}\n- $( sed 's/#/PR/' <<<"${pull_request_id}" ) ${pull_request_description}"
+    done
+
+    changelog_string_github="${changelog_string_github}\n#### ${repo_name}:\n$( echo -e "${changelog_strings_repo_github}" | sort -nk2 )\n"
+    changelog_string_deb="${changelog_string_deb}\n  * ${repo_name}:$( echo -e "${changelog_strings_repo_deb}" | sort -nk2 )"
+    changelog_string_yum="${changelog_string_yum}\n- ${repo_name}:$( echo -e "${changelog_strings_repo_yum}" | sort -nk2 )"
+
+    popd
+done
+
+# Write out a temporary Debian changelog with our new stuff appended and some templated formatting
+debian_changelog_file="deployment/debian-package-x64/pkg-src/changelog"
+debian_changelog_temp="$( mktemp )"
+# Create new temp file with our changelog
+echo -e "### DEBIAN PACKAGE CHANGELOG: Verify this file looks correct or edit accordingly, then delete this line, write, and exit.
+jellyfin (${new_version}-1) unstable; urgency=medium
+${changelog_string_deb}
+
+-- Jellyfin Packaging Team <packaging@jellyfin.org>  $( date --rfc-2822 )
+" >> ${debian_changelog_temp}
+cat ${debian_changelog_file} >> ${debian_changelog_temp}
+# Edit the file to verify
+$EDITOR ${debian_changelog_temp}
+# Move into place
+mv ${debian_changelog_temp} ${debian_changelog_file}
```

Comment:
Shouldn't you check that first comment line is gone? This allows to abort the process if changelog is bad.

Reply:
I'm actually on the fence about this one come to think of it. So the intention of these editors is just to get the person running it to give it a final once-over. Bailing out would leave us in no more undefined state than just letting it continue - at the end, the changes are only staged and not automatically committed, so if the whole process results in something bad, one can just `git reset --hard` to revert and try again.